src/scrapers/beinleumi.test.js

```diff
@@ -0,0 +1,52 @@
+import BeinleumiScraper from './beinleumi';
+import {
+  maybeTestCompanyAPI, extendAsyncTimeout, getTestsConfig, exportTransactions,
+} from '../../tests/tests-utils';
+import { SCRAPERS } from '../definitions';
+import { LOGIN_RESULT } from '../constants';
+
+const COMPANY_ID = 'beinleumi'; // TODO this property should be hard-coded in the provider
+const testsConfig = getTestsConfig();
+
+describe('Beinleumi', () => {
+  beforeAll(() => {
+    extendAsyncTimeout(); // The default timeout is 5 seconds per async test, this function extends the timeout value
+  });
+
+  test('should expose login fields in scrapers constant', () => {
+    expect(SCRAPERS.beinleumi).toBeDefined();
+    expect(SCRAPERS.beinleumi.loginFields).toContain('username');
+    expect(SCRAPERS.beinleumi.loginFields).toContain('password');
+  });
+
+  maybeTestCompanyAPI(COMPANY_ID, (config) => config.companyAPI.invalidPassword)('should fail on invalid user/password"', async () => {
```

Comment:
@dudiventura seems like you have redundant quotes sign
```suggestion
  maybeTestCompanyAPI(COMPANY_ID, (config) => config.companyAPI.invalidPassword)('should fail on invalid user/password', async () => {
```

Reply:
You're right,
Will delete it and push on the next commit.